csharp/HOLMS.Platform/HOLMS.Platform/Support/ReservationTags/TaxExemptTag.cs

```diff
@@ -6,6 +6,22 @@
 
 namespace HOLMS.Platform.Support.ReservationTags {
     public class TaxExemptTag : ReservationTagBase {
+        private string _taxExemptCategory;
+        private string _taxExemptID;
+
+        public TaxExemptTag(string [] tokens) {
+            if(tokens.Count() > 1) {
+                _taxExemptCategory = tokens[1];
+                _taxExemptID = tokens[2];
```

Comment:
I'm not the biggest fan of this constructor. I'm curious how you will use it. Will revisit this comment if needed.

Reply:
This is how ReservationTagBase builds the object from descriptors.